src/dewey/curation.clj

```diff
@@ -60,6 +60,10 @@
     (when (or (= type :data-object) (indexable? entity)) (op entity))
```

Comment:
This has nothing to do with your commit, but where is 'type' defined?  Is this supposed to be 'entity-type'?

Reply:
I think it is supposed to be entity-type. I wonder why it's not throwing any errors.